src/bundle/Resources/public/js/scripts/admin.contenttype.edit.js

```diff
@@ -617,5 +618,21 @@
         false,
     );
 
+    editForm.addEventListener(
+        'keydown',
+        (event) => {
+            const activeElementType = typeof doc.activeElement.type !== 'undefined' ? doc.activeElement.type.toLowerCase() : '';
+
+            if (event.key === ENTER_KEY && activeElementType !== 'textarea') {
```

Comment:
Why we want it work to every element except textarea?

Reply:
because in textarea you can use enter to go to a new line and not send the form, if there are still some excluded elements, I see no contraindications